services/gem/gem-rank.service.js

```diff
@@ -76,12 +76,22 @@ module.exports = class GemRank extends BaseJsonService {
     return [
       {
         title: 'Gems',
-        previewUrl: 'rt/puppet',
+        exampleUrl: 'rt/puppet',
+        placeholderUrl: 'rt/:package',
+        staticExample: this.render({
+          message: ordinalNumber(332),
```

Comment:
It seems like the call to `ordinalNumber` is a rendering concern and belongs in `render`. Perhaps the two parameters should be `count` and a field like `interval` or `period`.